### ISSUE TITLE
Long nasal bridge

### ISSUE THREAD

**Issue by jxchong** [obophenotype/human-phenotype-ontology#6199]
**Preferred term label:**
Long nasal bridge

**Synonyms**
Long nasal bridge, Long bridge of nose, Increased length of bridge of nose, Increased length of nasal bridge

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0000422

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1616-2448

**Comment by pnrobinson** [comment]
Just to confirm, are you referring to increased top-bottom length of the nasal bridge region?
This feature is found in OMIM:162100 ,  617755 ,  178110 , 108120  according to the CS.

**Comment by jxchong** [comment]
Yes I’m thinking it’s the opposite of Short nasal bridge HP:0003194

**Comment by pnrobinson** [comment]
added as HP:0033142, thanks for the suggestion!
